### ISSUE TITLE
selenium.common.exceptions.WebDriverException: Message: Can't load the profile.

### ISSUE THREAD

**Issue by 00h-i-r-a00** [SeleniumHQ/selenium#3745]
## Meta -
OS:  Ubuntu 14.01 (64-bit)

Selenium Version: 2.53.6  

Browser:  Firefox 

Browser Version:  47.0.1 (64-bit)

Python Version: 2.7.6
## Expected Behavior -
1. Open website 
2. Print url

## Actual Behavior -
Getting the error:

> 
> Traceback (most recent call last):
>   File "test.py", line 8, in <module>
>     driver = webdriver.Firefox(firefox_profile=profile)
>   File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/webdriver.py", line 80, in __init__
>     self.binary, timeout)
>   File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/extension_connection.py", line 52, in __init__
>     self.binary.launch_browser(self.profile, timeout=timeout)
>   File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/firefox_binary.py", line 68, in launch_browser
>     self._wait_until_connectable(timeout=timeout)
>   File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/firefox/firefox_binary.py", line 108, in _wait_until_connectable
>     % (self.profile.path))
> selenium.common.exceptions.WebDriverException: Message: Can't load the profile. Profile Dir: /tmp/tmp55V_vA/webdriver-py-profilecopy If you specified a log_file in the FirefoxBinary constructor, check it for details.
> 

## Steps to reproduce -

```
from selenium import webdriver
from xvfbwrapper import Xvfb

vdisplay = Xvfb()
vdisplay.start()

profile = webdriver.FirefoxProfile('/users/username/.mozilla/firefox/vhrv3vfz.Test')
driver = webdriver.Firefox(firefox_profile=profile)

driver.get('http://www.sohu.com')
print driver.current_url

vdisplay.stop()
driver.quit()
```
In addition to this I've `geckodriver 14` installed and saved in `/usr/local/bin`. Also, I am running this on a remote server, so need to run firefox headlessly. Note that the combination of `selenium 2.53.6` and `firefox 47.0.1` was running fine before, and just suddenly began giving this error. Its also extremely unpredictable, sometimes the error is given for all the websites that I am checking, and sometimes only a few. Would be great if anyone can help me with this, thanks!

**Comment by dborin** [comment]
Have you tried updating your Python Selenium package to 3.3.1?  If you need to maintain 2.53.6, try using a Python environment manager like pyenv or virtualenv to at least see if your issue is resolved in an updated package, and then you can work to get it updated in your test environment

**Comment by 00h-i-r-a00** [comment]
@dborin Thanks for the reply, and I apologize for not replying earlier. The issue got resolved by using Selenium 3.3.0 and firefox 52.0.2
